package.json

```diff
@@ -182,6 +164,7 @@
     "mockdate": "^3.0.5",
     "npm-run-all": "^4.1.5",
     "optimize-css-assets-webpack-plugin": "^6.0.1",
+    "patch-package": "^6.4.7",
```

Comment:
What is this used for?

Reply:
[`patch-package`](https://www.npmjs.com/package/patch-package) lets app authors instantly make and keep fixes to npm dependencies.

We've been using this package in the plugin ever since, it's not a new dependency. It was just moved from `dependencies` to `devDependencies`, which is good.